### PR TITLE
PR

### PR DESCRIPTION
**Pull request by RoongJin-APIPlus**

### CODE REVIEW

File: Pokedex.db
Comment (Nathavee-mh) [inline Pokedex.db:1]: add binary in git ignore

File: server.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	srv := handler.NewDefaultServer(graph.NewExecutableSchema(graph.Config{Resolvers: &graph.Resolver{
+		DB: graph.Database{DBPointer: db},
+	}}))
+
+	//DeletePokemon(db, 1, "Bulbasaur", "There is a plant seed on its back right from the day this Pokémon is born. The seed slowly grows larger.", "Seed", "Grass Poison", "Overgrow")
+	//FindPokemonById(db, 1)
```
Comment (Nathavee-mh) [inline server.go:59]: remove unused code

File: graph/resolver.go [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+		fmt.Println("Category: " + d3)
+		fmt.Println("Type: " + d4)
+		fmt.Println("Abilities: " + d5)
+	}
+
+	id_int64, _ := strconv.ParseInt(dummy, 10, 64)
+	return id_int64
+}
+
+func UpdatePokemon(db *sql.DB, id int, name string, description string, category string, typeOf []string, abilities []string) {
```
Comment (Nathavee-mh) [inline graph/resolver.go:60]: pekemonType?

**Review by Nathavee-mh — CHANGES_REQUESTED** [review]

**Review by Gromitmugs — APPROVED** [review]
excellent!

**Pull request closed without merging**